### PR TITLE
Appveyor: Build all targets

### PR DESCRIPTION
**Pull request by krocard**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/230%23discussion_r39491503%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/230%23issuecomment-140378450%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/230%23issuecomment-140378628%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/230%23issuecomment-140378450%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-15T12%3A50%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40OznOg%20approved%20informaly%22%2C%20%22created_at%22%3A%20%222015-09-15T12%3A51%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20f4a4d51a8a75e4fe8c8942835653fab91f3a4b35%20appveyor.yml%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/230%23discussion_r39491503%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20uses%20the%20%5C%22release%5C%22%20version%20of%20the%20libxml2%20for%20both%20debug%20and%20release%20builds.%22%2C%20%22created_at%22%3A%20%222015-09-15T09%3A40%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20appveyor.yml%3AL8-28%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull f4a4d51a8a75e4fe8c8942835653fab91f3a4b35 appveyor.yml 9'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/230#discussion_r39491503'>File: appveyor.yml:L8-28</a></b>
- <a href='https://github.com/dawagner'></a> this uses the "release" version of the libxml2 for both debug and release builds.
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/230#issuecomment-140378450'>General Comment</a></b>
- <a href='https://github.com/krocard'></a> :+1:
- <a href='https://github.com/krocard'></a> @OznOg approved informaly

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/230?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/230?approve=1'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/230'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/230'>Refresh</a>.<a href='#crh-update-none'></a>

File: appveyor.yml [hunk 1]
```diff
@@ -8,20 +8,20 @@ install:
 - wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - wget --no-check-certificate https://01.org/sites/default/files/asio-1.10.6.tar.gz
+- wget --no-check-certificate https://raw.github.com/philsquared/Catch/master/single_include/catch.hpp --directory-prefix catch
 - 7z x libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - 7z x libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - 7z x asio-1.10.6.tar.gz
 - 7z x asio-1.10.6.tar
+- set PREFIX_PATH=%APPVEYOR_BUILD_FOLDER%\libxml2-x86_64;%APPVEYOR_BUILD_FOLDER%\asio-1.10.6;%APPVEYOR_BUILD_FOLDER%\catch
```
Comment (dawagner) [inline appveyor.yml]: this uses the "release" version of the libxml2 for both debug and release builds.

**Comment by krocard** [comment]
:+1:

**Comment by krocard** [comment]
@OznOg approved informaly

**Comment by dawagner** [comment]
@krocard From what I understood, your are not suppsed to mix debug and release binaries. In this PR, you are building debug version of the parameter-framework against release version of libxml2.

**Comment by krocard** [comment]
@dawagner Mea culpa, I did not see the debug :( Will fix now.

**Pull request merged**